README.md

```diff
@@ -0,0 +1,29 @@
+# react-native-key-command
+react-native-key-command
+## Installation
+
+```sh
+npm install react-native-key-command
+```
+
+## Usage
+
+```js
+import { multiply } from "react-native-key-command";
+
+// ...
+
+const result = await multiply(3, 7);
+```
```

Comment:
Hmm, this can be changed later.

Reply:
This will be changed in upcoming PR's, so I won't bother updating this.